src/release/release-trigger.ts

```diff
@@ -126,10 +136,16 @@ export class ReleaseTrigger {
    */
   public readonly isContinuous: boolean;
 
+  /**
+   * Additional git push flags for manual releases
+   */
+  readonly gitPushFlags?: string[]
```

Comment:
Add modifier:

```suggestion
  public readonly gitPushFlags?: string[]
```

Also - who consumed this property?

Reply:
Done. Modifier added.